README.md

```diff
@@ -1 +1,2 @@
 # AppleCraft
+The name says it all.If you enjoy apples and minecraft then this is a mod for you! enjoy
```

Comment:
"The name says it all. If you enjoy apples and Minecraft then this is a mod for you! Enjoy!

Reply:
Fine